.circleci/config.yml

```diff
@@ -78,6 +83,38 @@ jobs:
       - run: { name: 'Release S3', command: "cd patternlab/styleguide && node_modules/.bin/gulp patternlab:release" }
       - run: { name: 'Release NPM', command: "cd patternlab/styleguide && node_modules/.bin/gulp npm:release" }
 
+  react_build_storybook:
+    <<: *react_defaults
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - mfr-dependencies-{{ checksum "react/package.json" }}
+
+      - run: cd react && npm install
+      - save_cache:
+          paths:
+            - react/node_modules
+          key: mfr-dependencies-{{ checksum "react/package.json" }}
+      - run:
+          name: Mayflower React Build
+          command: cd react && npm run build
+      - run:
+          name: Mayflower React Build Storybook
+          command: cd react && npm run build-storybook
+      - persist_to_workspace:
+          root: ~/repo
+          paths: .
+
+  react_deploy_tag:
+    <<: *react_defaults
+    steps:
+      - attach_workspace:
+          - at: ~/repo
+      - run: {name: 'Authenticate with registry', command: 'echo "//registry.npmjs.org/:_authToken=$npm_TOKEN" > ~/repo/react/.npmrc'}
+      - run: {name: 'Publish package', command: 'cd react && npm publish'}
+      - run: {name: 'Release S3', command: 'cd react && aws s3 sync storybook-static s3://mayflower-react.digital.mass.gov/ --delete'}
```

Comment:
currently for mayflower react s3, we also invalidate cache on cloudfront --> `aws cloudfront create-invalidation --distribution-id [!!PROD_DISTRIBUTION_ID_HERE!!] --paths '/*'` should we also do that here?